.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - phpunit-version: 4
+            php-version: '5.5'
+          - phpunit-version: 4
+            php-version: '5.6'
+          - phpunit-version: 5
+            php-version: '5.6'
+          - phpunit-version: 5
```

Comment:
initial solution was iterating over each PHP Version and each PHPUnit version, if they are compatible - run the tests.

now, we hardcoded the valid combinations, hoping we will never forget to add a valid variation.I think it's error-prone , but we also don't have so many changes in this repo, so not blocking

Reply:
If I understand the script correctly, it wasn't “if they are compatible”, but “if `composer require && composer update` does not fail”, which is also error-prone. I don't have any good idea how to do this check reliably.